packages/widget/src/components/address-input/address-input.ts

```diff
@@ -0,0 +1,84 @@
+import { LitElement, html } from 'lit';
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { Network } from '@buildwithsygma/sygma-sdk-core';
+import { when } from 'lit/directives/when.js';
+import { validateAddress } from '../../utils';
+import { styles } from './styles';
+
+@customElement('sygma-address-input')
+export class AddressInput extends LitElement {
+  static styles = styles;
+  @property({
+    type: String
+  })
+  address: string = '';
+
+  @property({ attribute: false })
+  onAddressChange: (address: string) => void = () => {};
+
+  @property({
+    type: String
+  })
+  network: Network = Network.EVM;
+
+  @state()
+  errorMessage?: string;
+
+  connectedCallback(): void {
+    super.connectedCallback();
+    if (this.address) {
+      this.handleAddressChange(this.address);
+    }
+  }
+
+  private handleAddressChange = (value: string): void => {
+    console.log('value', value);
+
+    if (this.errorMessage) {
+      this.errorMessage = undefined;
+    }
+
+    if (!value) {
+      return;
+    }
+
+    this.errorMessage = validateAddress(value, this.network);
+
+    if (!this.errorMessage) {
+      void this.onAddressChange(value);
+    }
+  };
+
+  render(): HTMLTemplateResult {
+    return html` <section class="inputAddressSection">
+      <div class="inputAddressContainer">
+        <label class="labelContainer">
+          <span>Send to </span>
+          ${when(
+            this.errorMessage,
+            () => html` <span class="errorMessage">${this.errorMessage}</span>`
+          )}</label
+        >
+        <textarea
```

Comment:
@mpetrunic yesterday after some discussion we arrive to one posible solution to the problem of not being able to make word-break for susbtrate address: tu use textarea. Minor changes here is that we need to trim the value before sending it to validation, since values of text area have a next line character

Reply:
Im fine with textarea

Reply:
The text area looks strange, especially since we can put the cursor in any place:

![Screenshot 2024-01-31 at 17 27 53](https://github.com/sygmaprotocol/sygma-widget/assets/8141917/bdb9c1bc-cd84-4fd9-9018-d4f13979eae6)

I didn't get the problem with using the input apart that the entire address value will not fit into the field. But from my experience, it's better than showing the first and last 4 characters f.e:
5E75.....8cem
Did I miss something?